### DISCUSSION TITLE
Add support for the yolov5 txt data format in mmdet and 
choose an algorithm for correctness verification. 给 mmdet 支持 yolov5 的 txt 数据格式，
并选择一个算法进行正确性验证

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#655] [MMDetection]
# Add support for the yolov5 txt data format in mmdet and 
choose an algorithm for correctness verification. 
 给 mmdet 支持 yolov5 的 txt 数据格式，
并选择一个算法进行正确性验证 

Support the widely used txt annotation format defined in yolov5 in mmdet, and choose an algorithm for correctness verification.

基于 yolov5 中定义的广泛使用的 txt 标注格式，在 mmdet 中支持该数据格式，并选择一个算法进行正确性验证

## Task Information

**Repository**：MMDetection

**Difficulty**：Medium

**Points**：30

**Estimated Time Consumption**：28 days

## Task Status

**Claimant's Github ID**：SimonGuoNjust

**Claim Time**：2023年10月31日

**PR Link**：None

## Points to Note

To claim the task, please reply `Claim the mission` in the comment section. After claiming the task, please add our assistant bot called MeowMewo on WeChat at: `openmmlabwx`.

After completing the task, please submit a PR with the title starting with [MMSIG]. After submission, please reply with your PR link in the comments section of this Discussion. We will review your work as soon as possible.

在评论区回复 `Claim the mission` 领取任务，领取任务后请及时添加小助手喵喵微信: `openmmlabwx`

完成任务后提交相关链接（如是PR，标题请以[MMSIG]开头）回复到本Disucssion评论区，我们会第一时间审核您的贡献。

**Comment by SimonGuoNjust** [comment]
Claim the mission
